src/app-layout/__tests__/utils.tsx

```diff
@@ -91,18 +92,9 @@ export function describeEachAppLayout(callback: () => void) {
 }
 
 export function isDrawerClosed(drawer: ElementWrapper) {
-  const element = drawer.getElement();
-
   // The visibility class name we are attaching to the wrapping element,
   // however the test-util points to the inner element, which has the scrollbar
-  if (element.parentElement!.classList.contains(styles['drawer-closed'])) {
-    return true;
-  }
-  // Apply the same logic for the visual refresh app-layout where the testutils selector is used.
-  if (element.parentElement!.classList.contains(testutilStyles['drawer-closed'])) {
-    return true;
-  }
-  return false;
+  return !!findUpUntil(drawer.getElement(), element => element.classList.contains(testutilStyles['drawer-closed']));
```

Comment:
Refactored this code to make one helper to apply to both refresh and classic versions